src/styles/blocks/container.scss

```diff
@@ -0,0 +1,21 @@
+.container {
+  --columm-count: 1;
```

Comment:
There is a typo in the custom property name `--columm-count`. It should be `--column-count` to correctly reflect the intended meaning.